src/nvim/globals.h

```diff
@@ -743,6 +743,7 @@ EXTERN bool km_startsel INIT( = false);
 EXTERN int cmdwin_type INIT( = 0);    ///< type of cmdline window or 0
 EXTERN int cmdwin_result INIT( = 0);  ///< result of cmdline window or 0
 EXTERN int cmdwin_level INIT( = 0);   ///< cmdline recursion level
+EXTERN bool cmdwin_lock INIT( = true);   ///< should lock for open cmdwin
```

Comment:
With my suggestion from earlier, might be better to make this `cmdwin_nolock_win`/`buf` instead (can bikeshed the naming), and only allow navigation between that window/buffer and the cmdwin?